src/preconnect.js

```diff
@@ -135,6 +145,20 @@ class Preconnect {
     return false;
   }
 
+  /** @private */
+  isPreloadSupported_() {
```

Comment:
So this is still only memoized per instance of `Preconnect`. Can we move it up into a script variable instead?